### DISCUSSION TITLE
Cannot “clear electricity price cap”

### DISCUSSION THREAD

**Discussion by haydnw** [BottlecapDave/HomeAssistant-OctopusEnergy#384] [Q&A]
We have a value entered in the Configuration box for “Optional electricity price cap in pence”, but now want to get rid of it. I’ve tried ticking the check box labelled “Clear electricity price cap” and clicking “SUBMIT”, at which point I’m told that options have been successfully saved. However, our dashboard doesn’t reflect any change, and when I go back into the integration configuration, the value is still in the text box. I’ve tried deleting the value from the box, and tried deleting the value *and* ticking the box. Each time it says that options have saved successfully but they never do. Does anyone have any suggestions please? Many thanks!

**Comment by anothermort** [comment]
It happened to me when I moved from ECO7 to Agile.
Had the same problems as you, so deleted the integration and readded it to HA.

**Comment by BottlecapDave** [comment]
Hi and sorry that you've encountered issues.

> However, our dashboard doesn’t reflect any change

The dashboard won't change for any data that has already been recorded. The only exception to this is for current consumption and you use the current consumption cost entity to represent the total costs. The change should occur for future calculations.

> when I go back into the integration configuration, the value is still in the text box.

This is odd. I've just tried this on my instance and it cleared the values successfully. Do you have any errors in your logs that might suggest why it's not occuring? The only other thing to suggest other than reinstalling the integration is set the cap to a value that will _hopefully_ never come about (like 500p).

**Comment by haydnw** [comment]
Thanks - a reboot hasn't done anything, and there's nothing appearing in the logs when I try to make the change. Updating the value to 500p as suggested has worked; not sure why it will save a new value but not clear it. Anyway, that'll do as a workaround for now, thanks. Could reinstall the integration but I have a number of automations and  related template sensors so don't really want to risk the names being different after reinstallation (I can't remember what I may have changed on original install!).
